### DISCUSSION TITLE
Troubleshooting hano/ditto integration?

### DISCUSSION THREAD

**Discussion by mmcc-xx** [eclipse-ditto/ditto#1613] [Q&A]
Following this as a model...
https://www.eclipse.org/packages/packages/cloud2edge/tour/

I've set up a bunch of Hano devices, Ditto things, and am attempting to integrate the two. I can post telemetry to the Hono device and it is accepted, but it does not show up in the corresponding Ditto thing.

Any suggestions for troubleshooting the problem?

**Comment by thjaeckle** [comment]
Please provide more information on the steps you took, api calls, on logs, on the IDs you used, etc.
